B827EBFFFEC843E6.json

```diff
@@ -0,0 +1,18 @@
+{
+ +  "gateway_conf": {
```

Comment:
Oh, now that the file has the proper extension, I notice that it has invalid stuff. All the `+` signs in front are copy&pasted from something else, please remove.